builders/get-started/networks/moonbeam-dev.md

```diff
@@ -90,6 +95,9 @@ To build the binary file, you can take the following steps:
     cd moonbeam
     ```
 
+    !!! note
+        Space in the installation file path will cause an compilation error.
```

Comment:
I am a little confused by this comment, do you mean if the user clones the repository in a directory that has a space in it? so the path is something like this?:

```
user/my documents/moonbeam
```

where `my documents` has a space in it?

Reply:
This is correct!